CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## [v5.0.0](https://github.com/stellar/js-stellar-base/compare/v4.0.3..v5.0.0)
```

Comment:
Can you restore the "Unreleased" heading above this, so we can collect new changes below it?

Reply:
I was going to do this after tagging the release so that its changelog was clean, but I'm realizing I'd need to make another PR for that lol. Good call, thanks!